main.cpp

```diff
@@ -576,40 +650,22 @@ bool MonitorController::toggleCrossfeed(bool status)
 int main()
 {
     Example::main();
-    Cat Pusya;
-    SipProvider sipnet;
     SpaceShip Proton;
-    PowerUnit powerunit;
-    DAC dacLeft;
-    
-    std::cout << "\nSIP trunk status:" << std::endl;
-    sipnet.printStatus ();
-    
-    std::cout << "\n";
-    powerunit.printStatus();
     
-    std::cout << "\nDAC status: " << dacLeft.checkError(440) << std::endl;
+    for ( int i = 2; i < 5; ++i ) Proton.makeLoop(i, 6);

```

Comment:
Even if your `if/else/for/while/etc` expressions have only one statement, you should still put it on its own line.
```
for( int i = 2; i < 5; ++i )
    proton.makeLoop(i, 6);
```
- [ ] revise